server/webhook_test.go

```diff
@@ -0,0 +1,88 @@
+package server
+
+import (
+	"context"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSendToWebhookIntegration(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
```

Comment:
Any reason for this?

Same for below.

Reply:
I want to differentiate between running unit tests and implementation tests. I looked up on how to accomplish this. This was, what came up as a go to solution.

Reply:
But this is an unit test from what I understand. There is no external dependency with any other service. It's purely inside the code.

Reply:
OK, removed.